drivers/metadata/postgres/metadata.go

```diff
@@ -77,29 +77,56 @@ func (r *metaReader) SetLimit(l int) {
 	r.limit = l
 }
 
+type Catalog struct {
+	Catalog          *metadata.Catalog
+	Owner            string
+	Encoding         string
+	Collate          string
+	Ctype            string
+	AccessPrivileges string
+}
+
+func (s Catalog) Values() []interface{} {
+	return []interface{}{s.Catalog.Catalog, s.Owner, s.Encoding, s.Collate, s.Ctype, s.AccessPrivileges}
+}
+
+func (s Catalog) GetCatalog() *metadata.Catalog {
+	return s.Catalog
+}
+
+var (
+	catalogsColumnName = []string{"Catalog", "Owner", "Encoding", "Collate", "Ctype", "Access privileges"}
+)
+
 func (r metaReader) Catalogs(metadata.Filter) (*metadata.CatalogSet, error) {
-	qstr := `
-SELECT d.datname as "Name"
+	qstr := `SELECT d.datname as "Name",
+       pg_catalog.pg_get_userbyid(d.datdba) as "Owner",
+       pg_catalog.pg_encoding_to_char(d.encoding) as "Encoding",
+       d.datcollate as "Collate",
+       d.datctype as "Ctype",
+       COALESCE(pg_catalog.array_to_string(d.datacl, E'\n'),'') AS "Access privileges"
 FROM pg_catalog.pg_database d`
 	rows, closeRows, err := r.query(qstr, []string{}, "1")
 	if err != nil {
 		return nil, err
 	}
 	defer closeRows()
 
-	results := []metadata.Catalog{}
+	var result []metadata.Result
```

Comment:
Here and where it's used:
```suggestion
	var results []metadata.Result
```